.github/workflows/build-windows.yml

```diff
@@ -0,0 +1,50 @@
+
+on:
+  workflow_call:
+    inputs:
+      operating-system:
```

Comment:
I think we can remove this from the Windows build

Reply:
I kept it because Github supports multiple versions of each OS. For Windows they have `windows-2022` and `windows-2019`: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#choosing-github-hosted-runners